Chapters/2-5-Chapter12.md

```diff
@@ -0,0 +1,175 @@
+
+Fanla stood just near a young campfire, looking out at the crowd. There were hundreds dotting the shore, huddled around multiple fires that had borrowed from the first. The bread supply had run out, so everyone was splitting pieces for each other, and huddling to keep warm. When Casra emerged from the forest, she immediately came to her and they shared a moment of pain together. Even from afar, Kipnu could tell there was something special about Fanla. In her eyes or her earnest sense of touch. She knew how to heal the thngis that he could not.
+
+Casra had to cut the moment short. This group of wayward travelers were looking for guidance that only thi could give, and every moment was more time for Shinna's guardians to gain knowledge of their presence and hae plan. Thi stood upon a small ridge in the sand and called out to the group, asking for quiet.
+
+"Mamen, Women, Men. My name is Casra, and I'm a doctor here on Ulukoma. What we've just seen and been through represents the dark side of the mainland. You are nothing to them; just a number on a manifest. But your lives are worth more than your past mistakes, more than your fear of the darkness."
+
+Every eye on that beach was fixed upon on Casra, searching for any sign that they would be safe.
+
+"I wish that I could tell you that things were different here..." thi paused to let a deep pain subside. "But there are some on this island that would shackle and drug every one of us to reap the fruit of _our_ labor, the work of _our_ hands. If you want to survive in this place, you will all have to fight for your freedom."
+
+A single pair of hands began to clap from a fire far away. It was a man, standing near a fire with the trees behind him, dressed in a thick black kelpen tunic with large rusted steel fasteners that seemed to be doubling as armor. Casra recognized him immediately.
+
+He spoke mockingly into the breeze as if he himself was king of Ulukoma. "Such drama, Casra. It's a shame your talents are wasted on an addict."
+
+A young maman dressed in a tight black robe tugged on his arm to stop. The light blue tint in hae eyes showed that thi was in the worst stages of withdraw. He disregarded hae, pushed hae away with ease and stepped closer to Casra.
+
+The man was on a ridge of his own now, speaking to the crowd. "Shinna grants the wishes of those who have earned their place in society. Wealth beyond what the mainland can offer, attention from the other sexes, euphoria that expands your reality. Now, ask yourself if you've earned it. If you're one of those vile criminals from the second boat, then by all means, follow Casra to hae hole in the ground. But if you're an upstanding member of society and you've come here to find a haven from the obscura, then come with me to the Great Pavilion. You deserve to be a part of the new world order."
```

Comment:
How does he know of the obscura? Do they have observation towers here? It's a deserted island, I assume with minimal communication from the rest of the world?